Jenkinsfile

```diff
@@ -29,13 +31,17 @@ pipeline {
   stages {
     stage('Lint') {
       steps {
-        sh "tox -e flake8"
+        sh "flake8"
       }
     }
     stage('Test') {
       steps {
         writeCapabilities(capabilities, 'capabilities.json')
-        sh "tox -e py27"
+        sh "pytest --junit-xml=results/py27.xml " +
+        "--html=results/py27.html " +
+        "--self-contained-html " +
+        "--log-raw=results/py27_raw.txt " +
+        "--log-tbpl=results/py27_tbpl.txt"
```

Comment:
Please indent lines 41-44 by two spaces. I also would like to remove the 'py27' component of these filenames, as we're no longer using this tox environment name. We can take care of the latter in a follow-up though.

Reply:
Fixed indentation.